packages/components/psammead-consent-banner/src/__snapshots__/index.test.jsx.snap

```diff
@@ -15,18 +15,35 @@ exports[`ConsentBanner should correctly render for ltr service 1`] = `
   }
 }
 
+@media (min-width:37.5rem) {
+  .emotion-0 {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+  }
+}
+
 .emotion-2 {
-  max-width: 80rem;
+  max-width: 37.4375rem;
   margin: 0 auto;
 }
 
-.emotion-2::after {
-  content: '\\0020';
-  display: block;
-  height: 0;
-  clear: both;
-  overflow: hidden;
-  visibility: hidden;
+.emotion-2 a {
+  color: #F6A21D;
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+  -webkit-text-decoration-color: #AEAEB5;
+  text-decoration-color: #AEAEB5;
+}
+
+.emotion-2 a:focus {
+  outline: none;
```

Comment:
Can you not use outline when you use box shadow? What's the reason for using box shadow instead? And why do something different here to the rest of the website? We use focus outline for everything to date.

Reply:
I used box-shadow because it's used here: https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/6000c4a406344f7bdcb17809

Happy to change to outline, but won't be able to get the same "fade" effect shown in the designs.

Reply:
We should always use the default outline. I believe the design was an 'illustration' of the default outline - Gail is going to make this clearer